frontend/src/views/StartView.vue

```diff
@@ -0,0 +1,217 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+import RadioButton from 'primevue/radiobutton';
+import { Breadcrumb, Button } from '@/lib/primevue';
+import { RouteNames } from '@/utils/constants';
+import type { Ref } from 'vue';
+
+// State
+const enquiryOrIntake: Ref<string | null> = ref(null);
+const acceptDisclaimer: Ref<string> = ref('');
```

Comment:
Consider just making this a boolean. You can then remove the function and on L120 just do `@click=acceptDisclaimer = !acceptDisclaimer`

Reply:
This is great! thanks!